src/noise.ts

```diff
@@ -76,6 +77,8 @@ export class Noise implements INoiseConnection {
     return {
       conn,
       remoteEarlyData: handshake.remoteEarlyData,
```

Comment:
shouldn't this be also removed?

There also also other places where earlyData is referenced

Reply:
Same comment as https://github.com/ChainSafe/js-libp2p-noise/pull/215#discussion_r982746339. Happy to do the breaking change here. Wdyt?

Reply:
I'll make this breaking change.